otlp/src/otlp/logs/logger.cpp

```diff
@@ -52,7 +52,9 @@ void AddAttribute(Item& item, std::string_view key, const Value& value) {
             [mutable_value](unsigned long long x) { mutable_value->set_int_value(x); },
             [mutable_value](float x) { mutable_value->set_double_value(x); },
             [mutable_value](double x) { mutable_value->set_double_value(x); },
-            [mutable_value](const std::string& x) { mutable_value->set_string_value(x); }},
+            [mutable_value](const std::string& x) { mutable_value->set_string_value(x); },
+            [mutable_value](const logging::JsonString& x) { mutable_value->set_string_value(std::string{x.Value()}); }
```

Comment:
Make `GetValue` return `const std::string&` to avoid a copy here

Reply:
`inline void AnyValue::set_string_value(std::string&& value)` rvalue is ok